CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.16.1
+	
+* Fixed a crashed caused by wrong numbers parsing of selector expressions
```

Comment:
This message could be a little more concrete, how about:

> Fixed a crash caused by parsing certain `calc()` expressions and variables names that contain numbers.

Reply:
Fixed